api/v1/openapi.yaml

```diff
@@ -1796,23 +1804,32 @@ definitions:
         description: MAC address
         type: string
       interface-name:
-        description: Name of network device
+        description: Name of network device in host netns
         type: string
       interface-index:
-        description: Index of network device
+        description: Index of network device in host netns
         type: integer
+      container-interface-name:
+        description: Name of network device in container netns
+        type: string
+      container-interface-is-secondary:
```

Comment:
Naming this "not primary" rather than "secondary", seems to make more sense. Not a blocker for me.